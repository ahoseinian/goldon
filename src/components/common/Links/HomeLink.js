import React from 'react'
import {Link} from 'react-router'
import {PrimaryButton} from '../Button'
import Icon from '../Icon'

export default() => (
  <Link to="/">
    <PrimaryButton><Icon name="home"/> <small>خانه</small></PrimaryButton>
  </Link>
)
