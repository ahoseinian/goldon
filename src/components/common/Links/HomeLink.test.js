import React from 'react'
import {shallow} from 'enzyme'
import HomeLink from './HomeLink'
import toJson from 'enzyme-to-json';

describe('HomeLink', () => {
  it('renders', () => {
    const wrapper = shallow(<HomeLink />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
