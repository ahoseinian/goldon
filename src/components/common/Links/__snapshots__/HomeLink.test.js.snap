exports[`HomeLink renders 1`] = `
<Link
  className="my-2"
  onlyActiveOnIndex={false}
  style={Object {}}
  to="/">
  <styled.button>
    <Icon
      name="home" />
    <span>
      خانه
    </span>
  </styled.button>
</Link>
`;
